i3lock.c

```diff
@@ -671,6 +682,8 @@ static void xcb_check_cb(EV_P_ ev_check *w, int revents) {
                         exit(0);
 
                     ev_loop_fork(EV_DEFAULT);
+                } else {
+                    maybe_close_xss_fd();
```

Comment:
nit: Is it necessary to only close the file descriptor in the non-forking case, or could we just unconditionally close the file descriptor?